test/tests/outputs/postprocessor/output_pps_hidden_shown_check.i

```diff
@@ -0,0 +1,90 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1
+  ymin = 0
+  ymax = 1
+  nx = 10
+  ny = 10
+  elem_type = QUAD4
+  # This test uses ElementalVariableValue postprocessors on specific
+  # elements, if you use DistributedMesh the elements get renumbered.
+  parallel_type = replicated
```

Comment:
The comments in the code said that with distributed mesh, elements get renumbered and therefore the PPS will look at the wrong place, which would make the test to diff.

Also, `parallel_type` takes 3 values: DISTRIBUTED, REPLICATED, DEFAULT. The default is `DEFAULT` and I am assuming it is related to how we configure libMesh and based on that it is either replicated or distributed. So, if we need to force the mesh distribution, we still need to have this flag on. That's my understanding...